modules/checking-mco-node-status.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-mco-node-status_{context}"]
+= Checking machine config update status
+
+During upgrades you might want to monitor the progress of individual nodes in the case that issues arise and you need to troubleshoot a node.
+
+To see the status of the Machine Config Operator (MCO) updates to your cluster, use the following `oc` commands:
+
+:FeatureName: Improved MCO state reporting
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+. Get a summary of update statuses for all nodes in a specific machine config pool by running the following command:
```

Comment:
below cmd `oc get machineconfignodes` will show all the MCNs. if you want to filter MCNs for specific MCP, you can run following cmd
```
$ oc get machineconfignodes $(oc get machineconfignodes -o json | jq -r '.items[]|select(.spec.pool.name=="worker")|.metadata.name')
```
replace `worker` with the pool name

Reply:
thank you, added this as a step